pkg/backend/display/testdata/not-truncated/stack-output-resource-diff.json.stdout.txt

```diff
@@ -0,0 +1,20 @@
+<{%reset%}>Configuration:<{%reset%}>
+<{%reset%}>  pulumi:pulumi:Stack: (same)
+<{%reset%}>    [urn=urn:pulumi:dev::pulumi-stack-output-diff::pulumi:pulumi:Stack::pulumi-stack-output-diff-dev]
+<{%reset%}><{%reset%}><{%!r(MISSING)eset%!}(MISSING)>    --outputs:--<{%!r(MISSING)eset%!}(MISSING)>
+<{%reset%}>    vehicles: <{%reset%}><{%reset%}>{
+<{%reset%}><{%reset%}>        taxi: <{%reset%}><{%reset%}>{
+<{%reset%}><{%reset%}>            color: <{%reset%}><{%reset%}>"yellow"<{%reset%}><{%reset%}>
+<{%reset%}><{%reset%}>            size : <{%reset%}><{%reset%}>"big"<{%reset%}><{%reset%}>
+<{%reset%}><{%reset%}>            urn  : <{%reset%}><{%reset%}>"urn:pulumi:dev::pulumi-stack-output-diff::some:demo:Taxi::taxi"<{%reset%}><{%reset%}>
+<{%reset%}><{%reset%}>        }<{%reset%}><{%reset%}>
+<{%reset%}><{%reset%}>        urn : <{%reset%}><{%reset%}>"urn:pulumi:dev::pulumi-stack-output-diff::some:demo:Vehicles::vehicles"<{%reset%}><{%reset%}>
+<{%reset%}><{%reset%}>    }<{%reset%}><{%reset%}>
+<{%reset%}>    <{%reset%}>  some:demo:Taxi: (same)
+<{%reset%}>        [urn=urn:pulumi:dev::pulumi-stack-output-diff::some:demo:Taxi::taxi]
+<{%reset%}><{%reset%}>    <{%reset%}>  some:demo:Vehicles: (same)
+<{%reset%}>        [urn=urn:pulumi:dev::pulumi-stack-output-diff::some:demo:Vehicles::vehicles]
```

Comment:
Printing the stack outputs after completion of what the test simulates is expected behaviour – the fix introduced by the patch is perhaps easier to grok by looking at `pkg/backend/display/testdata/not-truncated/webserver-userdata.json.stdout.txt`.